BlackJackets API/src/main/java/BlackJackets/BlackJackets/dto/VenueDto.java

```diff
@@ -21,7 +21,7 @@
 public class VenueDto {
 
         @Id
-        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        @GeneratedValue
```

Comment:
Not sure if this should have been changed, but we can change it back afterewards